CanvasColorSpaceProposal.md

```diff
@@ -1,52 +1,33 @@
-
-An aspect of current implementations that is interoperable is that colors match between CSS/HTML and canvases:
-* A color value used as a canvas drawing style will have the same appearance as if the same color value were used as a CSS style
-* An image resource drawn to a canvas element will have the same appearance as if it were displayed as the replaced content of an HTML element or used as a CSS style value.
-
-This color matching behavior needs to be preserved to avoid breaking pre-existing content.
-
-Some implementations convert images drawn to canvases to the sRGB color space. This has the advantage of making the color correction behavior device independent, but it clamps the gamuts of the rendered content to the sRGB gamut, which is significantly narrower than the gamuts of some current consumer devices.
+* The color space of canvases is undefined in the current specification, although is de facto sRGB.
+* The bit-depth of canvases is likewise undefined, although is de facto 8 bits per component.
+* Images retrieved from a canvas are in an undefined color space and no color space information is encoded by ``toDataURL`` or ``toBlob``.
 
 ### Requests for this Feature
 
 * <cite>[https://github.com/whatwg/html/issues/299]</cite> <blockquote><p>Allow 2dcontexts to use deeper color buffers</p></blockquote>
 * <cite>[https://bugs.chromium.org/p/chromium/issues/detail?id=425935]</cite> <blockquote><p>Wrong color profile with 2D canvas</p></blockquote>
-* Engineers from the Google Photos, Maps and Sheets teams have expressed a desire for canvases to become color managed.  Particularly for the use case of resizing an image, using a canvas, prior to uploading it to the server, to save bandwidth. The problem is that the images retrieved from a canvas are in an undefined color space and no color space information is encoded by toDataURL or toBlob.
+* Engineers from the Google Photos, Maps and Sheets teams have expressed a desire for canvases to be color managed. Particularly for the use case of resizing an image, using a canvas, prior to uploading it to the server, to save bandwidth. The problem is that the images retrieved from a canvas are in an undefined color space and no color space information is encoded by toDataURL or toBlob.
 
-## Background
 
-Color spaces are generally tuples of:
-* Chromaticity coordinates (X,Y in CIE 1931 space)
-  * Primaries (coords for 1.0 for each of R/G/B)
-  * White-point (coord for white)
-* Transfer function (often "gamma")
 
-Within the same chromaticity, 0.0 will always be min-brightness and 1.0 is max-brightness.
-However, because of the transfer function, 0.5 will not be the average physical brightness of 0.0 and 1.0.
-You can see this by making stripes of 1.0 and 0.0, and matching that pattern's brightness to a solid grey background.
-(On my machine, striped 0/1/0/1 appears equally bright as #ACACAC, or 0.67)
```

Comment:
This is really useful background that I would like to remain visible here, but I'm happy to collaborate on better wording.

Reply:
I agree that some background will be helpful, but I'd prefer to leave it out for now, because (1) it's redundant with the CSS Color Level 4 spec, which specifies the color spaces and (2) I'd like to get behavior and API changes nailed down. We can pull this out of the revision history as a starting point when we get there.